### PR TITLE
Devel gst

### PR DESCRIPTION
**Pull request by afropapp13**
Adding xsec, differential xsec & phase space choice in the gst file.

### CODE REVIEW

File: src/Apps/gNtpConv.cxx [hunk 1]
```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
                                  //  - ((e/h) * energy)   for pi0, gamma, e-, e+, where e/h is set to 1.3
                                  //  - (kinetic energy) for other particles
 
+  double brXSec;              // the event cross section in 1E-38cm^2
+  double brDXSec;             // is the differential cross section for the selected in 1E-38cm^2/{K^n}
+  int    brKPS;               // phase space that the xsec has been evaluated into as defined in
+                              // https://github.com/afropapp13/Generator/blob/master/src/Framework/Conventions/KinePhaseSpace.h
```
Comment (mroda88) [inline src/Apps/gNtpConv.cxx:404]: can we get rid of this link?

File: src/Apps/gNtpConv.cxx [hunk 2]
```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
                                  //  - ((e/h) * energy)   for pi0, gamma, e-, e+, where e/h is set to 1.3
                                  //  - (kinetic energy) for other particles
 
+  double brXSec;              // the event cross section in 1E-38cm^2
+  double brDXSec;             // is the differential cross section for the selected in 1E-38cm^2/{K^n}
+  int    brKPS;               // phase space that the xsec has been evaluated into as defined in
```
Comment (mroda88) [inline src/Apps/gNtpConv.cxx:403]: I would prefer this to be an `unsigned int`. So, I would prefer this to be `UInt_t`

File: src/Apps/gNtpConv.cxx [hunk 3]
```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
                                  //  - ((e/h) * energy)   for pi0, gamma, e-, e+, where e/h is set to 1.3
                                  //  - (kinetic energy) for other particles
 
+  double brXSec;              // the event cross section in 1E-38cm^2
```
Comment (mroda88) [inline src/Apps/gNtpConv.cxx:401]: Following https://root.cern.ch/doc/master/classTTree.html I would recommend to define these variables as Double_t

File: src/Apps/gNtpConv.cxx [hunk 4]
```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
...
+  double brDXSec;             // is the differential cross section for the selected in 1E-38cm^2/{K^n}
```
Comment (mroda88) [inline src/Apps/gNtpConv.cxx:402]: same here

File: src/Apps/gNtpConv.cxx [hunk 5]
```diff
@@ -502,6 +507,9 @@ void ConvertToGST(void)
   s_tree->Branch("vtxt",         &brVtxT,	    "vtxt/D"        );
   s_tree->Branch("sumKEf",       &brSumKEf,	    "sumKEf/D"      );
   s_tree->Branch("calresp0",     &brCalResp0,	    "calresp0/D"    );
+  s_tree->Branch("XSec",         &brXSec,	    "XSec/D"    );
+  s_tree->Branch("DXSec",         &brDXSec,	    "DXSec/D"    );
+  s_tree->Branch("KPS",          &brKPS,	    "KPS/I"    );
```
Comment (mroda88) [inline src/Apps/gNtpConv.cxx:512]: Make it an unsigned int, `"KPS/i"`

**Review by mroda88 — CHANGES_REQUESTED** [review]
There are minor changes, but the fix seems trivial to me. 
@candreop are you ok with adding cross sections into the gst tree?

**Comment by candreop** [comment]
Sounds ok @mroda88. Though I am sure that sooner or later we will see a github issue with a plot of the differential cross-section, straight out of gst, from a user who is not aware that, say, the stored values for resonance and DIS events are not comparable (without a jacobian)

**Review by mroda88 — APPROVED** [review]
It's good for me

**Pull request merged**